src/test/java/de/tum/cit/aet/artemis/core/util/CourseUtilService.java

```diff
@@ -195,22 +195,12 @@ public class CourseUtilService {
     private GradingScaleUtilService gradingScaleUtilService;
 
     /**
-     * Creates and saves a course with null id.
+     * Creates and saves a course with a new id (automatically generated).
```

Comment:
I think this would be more clear, I am not sure if we need the comment at all
```suggestion
     * Creates and saves a course (`id` is automatically generated).
```